views/table_chart_plugin_library_selector.inc

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Contains table_chart_plugin_library_selector.
+ */
+
+class table_chart_plugin_library_selector extends views_plugin_display_extender {
```

Comment:
I know views class names use underscores (and a lot of our old views classes do too). But we should use the proper Cased class names are per the updated Drupal coding spec.